### PR TITLE
Phase2-hgx319C Modify the test scrpts to check if all wafers in the flat file are in the list of valid modules

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify the test scrpts to check if all wafers in the flat file are in the list of valid modules

#### PR validation:

Use the scripts in the test directory of Geometry/HGCalGeometry

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

File: Geometry/HGCalGeometry/test/python/testHGCalWaferInFileCheck_cfg.py [hunk 1]
```diff
@@ -1,22 +1,50 @@
...
+                  "geometry of operations: D83, D88, D92, D93")
+
+### get and parse the command line arguments
+options.parseArguments()
+print(options)
+
+####################################################################
+# Use the options
+if (options.geometry == "D92"):
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
```
Comment (srimanob) [inline Geometry/HGCalGeometry/test/python/testHGCalWaferInFileCheck_cfg.py:28]: Why duplications of
```
from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
process = cms.Process('WaferInFileCheck',Phase2C11M9)
```
in all conditions?

File: Geometry/HGCalGeometry/test/python/testHGCalWaferTypeTester_cfg.py [hunk 2]
```diff
@@ -1,14 +1,53 @@
...
+                  "geometry of operations: D83, D88, D92, D93")
+
+### get and parse the command line arguments
+options.parseArguments()
+print(options)
+
+####################################################################
+# Use the options
+if (options.geometry == "D92"):
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
```
Comment (srimanob) [inline Geometry/HGCalGeometry/test/python/testHGCalWaferTypeTester_cfg.py:28]: Same question as above.

**Comment by srimanob** [comment]
+Upgrade

I assume what I comment is OK to follow up later, as no comment back from @bsunanda

**Comment by qliphy** [comment]
+1
[Further comments](https://github.com/cms-sw/cmssw/pull/38847#discussion_r928654561) to be followed by @bsunanda

**Pull request merged**